src/build/content/server.ts

```diff
@@ -105,6 +105,7 @@ export const copyNextServerCode = async (ctx: PluginContext): Promise<void> => {
         `server/*`,
         `server/chunks/**/*`,
         `server/edge-chunks/**/*`,
+        `server/edge/chunks/**/*`,
```

Comment:
needed for edge runtime pages when building with turbopack

`server/edge-chunks` seems to contain things like assets and wasm files (?) for middleware for non-turbopack builds